### PR TITLE
update server to work with new mongo-go-driver/bson changes

### PR DESCRIPTION
**Pull request by Bastiantheone**
There have been some changes to github.com/mongodb/mongo-go-driver/bson and the current code doesn't work anymore.

### CODE REVIEW

**Comment by simplesteph** [comment]
LGTM!

File: blog/blog_server/server.go [hunk 1]
```diff
@@ -84,7 +84,7 @@ func (*server) ReadBlog(ctx context.Context, req *blogpb.ReadBlogRequest) (*blog
 
 	// create an empty struct
 	data := &blogItem{}
-	filter := bson.NewDocument(bson.EC.ObjectID("_id", oid))
+	filter := bson.M{"_id": oid}
```
Comment (dbalencar) [inline blog/blog_server/server.go:87]: why not **filter := primitive.M{"_id": oid}** instead? bson.M is from another package.

File: blog/blog_server/server.go [hunk 2]
```diff
@@ -121,7 +121,7 @@ func (*server) UpdateBlog(ctx context.Context, req *blogpb.UpdateBlogRequest) (*
...
```
Comment (dbalencar) [inline blog/blog_server/server.go:124]: why not filter := primitive.M{"_id": oid} instead? bson.M is from another package

File: blog/blog_server/server.go [hunk 3]
```diff
@@ -152,15 +152,15 @@ func (*server) UpdateBlog(ctx context.Context, req *blogpb.UpdateBlogRequest) (*
...
+	oid, err := primitive.ObjectIDFromHex(req.GetBlogId())
 	if err != nil {
 		return nil, status.Errorf(
 			codes.InvalidArgument,
 			fmt.Sprintf("Cannot parse ID"),
 		)
 	}
 
-	filter := bson.NewDocument(bson.EC.ObjectID("_id", oid))
+	filter := bson.M{"_id": oid}
```
Comment (dbalencar) [inline blog/blog_server/server.go:163]: why not filter := primitive.M{"_id": oid} instead? bson.M is from another package

**Pull request merged**